subprojects/diagnostics/src/main/java/org/gradle/api/tasks/diagnostics/internal/dependencies/AsciiDependencyReportRenderer.java

```diff
@@ -112,6 +113,12 @@ void renderNow(RenderableDependency root) {
     @Override
     public void complete() {
         legendRenderer.printLegend();
+
+        getTextOutput().println();
```

Comment:
❌ I just realized that we now always print this, even when the user is already using the scan plugin. We should not bother them in that case.